stereo_image_proc/test/test_point_cloud_node.py

```diff
@@ -68,12 +68,28 @@ def generate_test_description():
             ],
             output='screen'
         ),
-        # PointCloudNode
+        # PointCloudNode (color enabled)
         Node(
             package='stereo_image_proc',
             executable='point_cloud_node',
             name='point_cloud_node',
-            output='screen'
+            output='screen',
+            parameters=[
+                {'use_color': True}
+            ],
+        ),
+        # PointCloudNode (color disabled)
+        Node(
+            package='stereo_image_proc',
+            executable='point_cloud_node',
+            name='point_cloud_node',
```

Comment:
I would give this node a different name (or a namespace) to differentiate it from the other. If we use a namespace, then we can probably remove the remappings  argument below since the `points2` topic should appear in the namespace.

Reply:
I renamed the node, but didn't add a namespace as remappings would be required either way.